### ISSUE TITLE
Bode throws error on Init while Raginator is active in DNDB

### ISSUE THREAD

**Issue by trey-kirk-sp** [mr-ice/maptool-macros#39]
SWITCH option found no match for "advantage".       Statement options (if any): h, switch (value)       Statement Body : case "expertise": tempAbilityValue = 3; case "proficiency": tempAbilityValue = 2; case "half-proficiency": tempAbilityValue = 1

[Bode-mrice-error.rptok.zip](https://github.com/mr-ice/maptool-macros/files/4795745/Bode-mrice-error.rptok.zip)

**Comment by trey-kirk-sp** [comment]
Doesn't happen in dndBeyond_issue17. Might have accidentally fixed it already.

**Comment by trey-kirk-sp** [comment]
Might be the place to expand how Advantage / Disadvantage modifiers are modeled and getting the macros to look for them.

**Comment by trey-kirk-sp** [comment]
As of hash a4fe2431, no longer reproduces. Could not force an error with a fresh token or with the captured token from the game session. Looks like I fixed it in hash df889107 as part of Issue #15 and added a default case:

```
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 59) <!-- Now tease out the interesting bits -->
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 60) <!-- No need to keep searching for stuff. This arry shouldnt be very big -->
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 61) [h: abilityValue = 0]
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 62) [h, foreach (abilityMod, abilityMods), code: {
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 63)        [h: value = json.get (abilityMod, "type")]
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 64)        [h, switch (value):
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 65)                case "expertise": tempAbilityValue = 3;
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 66)                case "proficiency": tempAbilityValue = 2;
df889107 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-15 14:52:12 -0500 67)                case "half-proficiency": tempAbilityValue = 1;
df889107 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-15 14:52:12 -0500 68)                default: tempAbilityValue = 0]
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 69)        [h: abilityValue = math.max (abilityValue, tempAbilityValue)]
0d202266 Lib-DNDBeyond/dndb_getSkill.command (trey-kirk-sp 2020-06-13 19:41:39 -0500 70) }]
```
